app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,54 @@
+<section class="food_list_container flex mx-8 py-4 items-center justify-center mt-24">
+  <div class="food flex flex-col items-centre gap-2">
+    <%if @recipes.empty?%>
+        <h2>Oops!! Recipe List is Empty</h2>
+        <%= link_to 'Add Recipe', new_recipe_path(), class: "btn" %>
+    <%else%> 
+      <div class="recipe_title flex flex-col gap-4">
+        <article class="top flex justify-center text-center">
+         
+          <div class="mb-4 grid">
+            <%= link_to 'Add Food', new_food_path(), class: "btn btn-primary justify-self-start" %>
+          </div> 
+          <h2 class="text-3xl font-extrabold leading-none tracking-tight text-gray-900 md:text-5xl lg:text-6xl dark:text-white">Find your favourite recipe here</h2>
+        </article>
+        <div class="p-2 flex flex-col gap-2">
+          <% @recipes.each_with_index do |recipe, index| %>
+            <div class="recipe_class flex flex-col gap-6">
+              <div class="des grid grid-cols-4 gap-2">
+                <div>
+                  <h2 class="card-title text-xl font-semibold text-gray-900"><%= link_to "#{recipe.name}", recipe %></h2>
+                  <p class="c"><strong>Preparation Time:</strong> <%= recipe.preparation_time %> (mins)</p>
+                  <p class="c"><strong>Cooking Time:</strong> <%= recipe.cooking_time %> (mins)</p>
+                </div>
```

Comment:
- [ ] On this page, there are two things that needs to be removed as I pointed in the image:
      - The Add Food button should not be here at all
      - The delete button is actually not active because the usert was not the one that added it, GOOD JOB!!! But we should also not show the button at all but the color of the button still shows, kindly remove it too.
![image](https://github.com/fatmahussein/Recipe-app/assets/86200356/4943cd4c-5829-4ef3-81d2-1ba3cc49a2ed)